package.json

```diff
@@ -1,5 +1,29 @@
 {
+  "name": "@gomaestro/maestro-js",
+  "version": "0.0.0",
```

Comment:
This is mentioned as `0.0.0` and once it is merged, changeset will generate a PR to bump this to `0.1.0`.
Also @Vardominator, you mentioned that you would like version to be `1.0.0` however, current api can't be said to be stable, and there might be breaking (backward incompatible) changes which is communicated by incrementing major (thus your `1.0.0` may become `2.0.0` very soon). However, when major is 0, then we can include breaking changes in subsequent minor versions.